app/models/question.rb

```diff
@@ -85,25 +66,7 @@ class Question < ApplicationRecord
             AND questionings.type = 'Questioning'
         LEFT OUTER JOIN forms ON forms.deleted_at IS NULL AND forms.id = questionings.form_id
         WHERE inner_questions.deleted_at IS NULL AND inner_questions.id = questions.id
-    ) AS form_published_col").
-    select("(
-      SELECT COUNT(DISTINCT copy_answers.id)
-        FROM questions inner_questions
-        LEFT OUTER JOIN form_items questionings
-          ON questionings.deleted_at IS NULL AND questionings.question_id = inner_questions.id
-            AND questionings.type = 'Questioning'
-        LEFT OUTER JOIN forms ON forms.deleted_at IS NULL AND forms.id = questionings.form_id
-        LEFT OUTER JOIN answers ON answers.deleted_at IS NULL AND answers.questioning_id = questionings.id
-        LEFT OUTER JOIN questions copies ON questions.deleted_at IS NULL AND questions.is_standard = true
-          AND questions.id = copies.original_id
-        LEFT OUTER JOIN form_items copy_questionings ON copy_questionings.deleted_at IS NULL
-          AND copy_questionings.question_id = copies.id AND copy_questionings.type = 'Questioning'
-        LEFT OUTER JOIN forms copy_forms ON copy_forms.deleted_at IS NULL
-          AND copy_forms.id = copy_questionings.form_id
-        LEFT OUTER JOIN answers copy_answers ON copy_answers.deleted_at IS NULL
-          AND copy_answers.questioning_id = copy_questionings.id
-        WHERE inner_questions.deleted_at IS NULL AND inner_questions.id = questions.id
-    ) AS copy_answer_count_col")
+    ) AS form_published_col")
```

Comment:
How come this chunk survived the massacre?

Reply:
because the method was still being called, and I wasn't sure if just returning questions was the right approach. let's discuss.